File: src/main/java/gestion_restaurant/entity/CommandeMenuComplement.java
package gestion_restaurant.entity;

import lombok.*;
import java.math.BigDecimal;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class CommandeMenuComplement {
    private Long id;
    private CommandeItem commandeItem;
    private Complement complement;
    private Integer quantite;
    private BigDecimal prixUnitaire;
    private String remarque;
}

File: Models/CommandeMenuComplement.cs
namespace gestion_restaurant.Models
{
    public class CommandeMenuComplement
    {
        public long Id { get; set; }

        public long? CommandeItemId { get; set; }
        public CommandeItem CommandeItem { get; set; }

        public long? ComplementId { get; set; }
        public Complement Complement { get; set; }

        public int Quantite { get; set; }
        public decimal? PrixUnitaire { get; set; }
        public string Remarque { get; set; }
    }
}
